omod/src/main/webapp/resources/htmlforms/HMIS-HIV-003-HivCareArtCard-ClinicalAssessmentPage.xml

```diff
@@ -913,581 +1133,1838 @@
-                                            <div class="row">
-                                                <div class="col-md-2">
-                                                    <label>Advanced Disease Status</label>
-                                                </div>
-                                                <div class="col-md-3">
-                                                    <obs conceptId="165272" class="long-value-select" answerConceptIds="165309, 165266, 165267, 165268, 165269, 165270, 165271"
-                                                         answerLabels="1 - No Adv. Disease, 2 - Suspected Adv. Disease, 3a - Confirmed Adv. Disease CD4 &lt; 200, 3b - Confirmed Adv. Disease - Pos CrAg,
-                                                         3c - Confirmed Adv. Disease - Pos TB, 3d - Confirmed Adv. Disease - WHO Stage 3/4, 3e - Confirmed Adv. Disease - Child &lt; 5 Yrs"/>
-                                                </div>
-                                                <div class="col-md-1"></div>
-                                                <div class="col-md-2">
-                                                <label>WHO Clinical stages</label>
+                                        <div class="card" style="margin-top:10px">
+                                            <div class="card">
+                                                <div class="card-header">
+                                                    <label>OTHER MEDICATIONS</label>
                                                 </div>
-                                                <div class="col-md-3">
-                                                    <obs id="90203" class="long-value-select" conceptId="90203" answerConceptIds="90033,90034,90035,90036,90293,90294,90295,90296" answerLabels="1,2,3,4,T1,T2,T3,T4"/>
+                                                <div class="card-body">
+                                                    <div>
+                                                        <repeat>
+                                                            <template>
+                                                                <obsgroup groupingConceptId="160741">
+                                                                    <div class="row multi-drug"
+                                                                         id="{id}-toggleContainer">
+                                                                        <div class="col-2">
+                                                                            <obs labelText="Drug" conceptId="1282"
+                                                                                 answerConceptSetIds="1157,1158,1159,1195,1197,5279,5839"
+                                                                                 style="autocomplete"
+                                                                                 class="fullwidth"/>
```

Comment:
@slubwama What does the class fullwidth do? It is not a boostrap class

Reply:
done